### PR TITLE
[TS] LPP-49065>LPS-185695 Display timing of duplicate message of friendly URL is strange when web content is added via widget

### PR DESCRIPTION
**Pull request by ces-quanhuynh**
Hi @locpham97
Please review this PR and leave your comment.
Thanks,
Quan 
# Motivation
This PR is for fixing [LPS-185695](https://issues.liferay.com/browse/LPS-185695)
# Issue
The Asset Publisher widget does not have the Toast to display the warning from MultiSessionMessage. 
When send the MultiSessionMessage, only the Web Content can receive it and display. 
That why when adding WC by Asset Publisher Widget, there is no warning until we access to Web Content.
# Proposed Solution
Add the Toast to display warning in the Asset Publisher which is similar to the Toast in Web Content. Then add the key to distinguish when the WC created by Asset Publisher Widget or in the Web Content to display the warning in correct time and place.

### CODE REVIEW

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -163,4 +167,16 @@ SearchContainer<AssetEntry> searchContainer = assetPublisherDisplayContext.getSe
...
-</aui:script>
\ No newline at end of file
+</aui:script>
+
+<c:if test="<%= Validator.isNotNull(friendlyURLChangedMessage) %>">
+	<aui:script>
+		Liferay.Util.openToast({
+			message: '<%= friendlyURLChangedMessage %>',
+			toastProps: {
+				autoClose: 20000,
```
Comment (locpham97) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp]: I think it is better to change it to 5000

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/UpdateArticleMVCActionCommand.java [hunk 2]
```diff
@@ -522,8 +522,14 @@ else if (group.hasStagingGroup()) {
 			return Collections.emptyMap();
 		}
 
+		String portletResource = ParamUtil.getString(
+			actionRequest, "portletResource");
+
+		boolean isPortlet = !portletResource.isEmpty();
```
Comment (locpham97) [inline modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/UpdateArticleMVCActionCommand.java]: using the !Validator.isBlank or Validator.isNotNull(), write :
`

		boolean isPortlet = !Validator.isBlank(ParamUtil.getString(
			actionRequest, "portletResource"));`

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp [hunk 3]
```diff
@@ -1,3 +1,5 @@
+<%@ page import="com.liferay.portal.kernel.servlet.MultiSessionMessages" %>
```
Comment (locpham97) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp]: add in init.jsp

Comment (ces-quanhuynh) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp]: I've fixed this.

File: modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/UpdateArticleMVCActionCommand.java (hunk 2 above)
Comment (ces-quanhuynh) [reply modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/portlet/action/UpdateArticleMVCActionCommand.java]: I've fixed this.

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp (hunk 1 above)
Comment (ces-quanhuynh) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/view.jsp]: I think we should follow the time set in Web Content https://github.com/liferay/liferay-portal/blob/6221e884ae62e693f52fb358482185dd16383216/modules/apps/journal/journal-web/src/main/resources/META-INF/resources/friendly_url_changed_message.jspf#L42

**Comment by ces-quanhuynh** [comment]
@locpham97 
Thank you for your review.
I've updated the code and leave some comment.
Please take a look.

**Comment by locpham97** [comment]
LGTM

**Comment by locpham97** [comment]
See https://github.com/liferay-lima/liferay-portal/pull/4344

**Pull request closed without merging**